package.json

```diff
@@ -24,7 +24,6 @@
     "peasant": "^0.5.2"
   },
   "dependencies": {
-    "babel-runtime": "^5.0.0",
-    "slug": "^0.9.1"
+    "babel-runtime": "^5.0.0"
```

Comment:
I don't see any usage of this library. Maybe we can simply remove this one as well?